coalib/bears/Bear.py

```diff
@@ -316,9 +316,9 @@ def setup_dependencies():
         """
 
     @classmethod
-    def check_prerequisites(cls):
+    def check_all_deps(cls):
```

Comment:
actually no, check_all_deps later calls check_prequisites and bears can still use that :D